### PR TITLE
replace `handleWithGlobalMiddlewares` with `handlerWithGlobalMiddlewa…

### PR DESCRIPTION
**Pull request by gorilla001**
**replace `handleWithGlobalMiddlewares` with `handlerWithGlobalMiddlewa**

**\- A picture of a cute animal (not mandatory but encouraged)**

…res`

Signed-off-by: mgniu mgniu@dataman-inc.com

### CODE REVIEW

File: api/server/middleware.go [hunk 1]
```diff
@@ -9,7 +9,7 @@ import (
 // handleWithGlobalMiddlwares wraps the handler function for a request with
```
Comment (tonistiigi) [inline api/server/middleware.go]: change the comment as well

**Comment by cpuguy83** [comment]
ping

**Comment by cpuguy83** [comment]
LGTM

**Comment by tonistiigi** [comment]
LGTM

**Pull request merged**